dist/Exporter/Makefile.PL

```diff
@@ -9,7 +9,7 @@ WriteMakefile(
     VERSION_FROM => 'lib/Exporter.pm',
     ( $] > 5.011 ) ? () : ( INSTALLDIRS => 'perl' ),    # CPAN sourced versions should now install to site
     PREREQ_PM => {
-        'Carp' => '1.05',
+        'Carp' => '1.50',
```

Comment:
Carp 1.50 was first released with perl 5.28 (see [perl5280delta](https://perldoc.perl.org/perl5280delta#Updated-Modules-and-Pragmata)). This change would mean that Exporter now requires perl 5.28+ when previously it worked fine on 5.8. I don't think that's acceptable.

Reply:
Carp is dual-life. That said, there is no reason for this change.